Structure_Engine/Query/Normal.cs

```diff
@@ -66,7 +66,7 @@ public static List<Vector> Normals(this FEMesh mesh)
         [Output("normal", "Vector representing the local z-axis of a mesh face.")]
         public static Vector Normal(this FEMeshFace face, FEMesh mesh)
         {
-            if (!face.NullCheck(mesh, "Normal"))
+            if (face.IsNull(mesh, "Normal"))
```

Comment:
here I do not think providing method name is required, as it is the last argument.